.gitignore

```diff
@@ -1,5 +1,6 @@
 database.db
 *pyc
+books/migrations/*
```

Comment:
Awesome@ agree models are something to hopefully take a look at sometime soon. :)